entrypoint.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh -l
+set -euo pipefail
+
+echo "setting prerequisites."
+export BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr -d '[:punct:]')
+
+
+if [ $BRANCH_NAME != 'master' ];
```

Comment:
Can you use gitversion to get the version?

Reply:
Changed to use gitversion.